pytest.ini

```diff
@@ -1,4 +1,8 @@
 [pytest]
 testpaths = test/ kolibri/
-DJANGO_SETTINGS_MODULE = kolibri.deployment.default.settings.test
 norecursedirs = dist tmp* .svn .*
+# Settings for pytest-django
+django_find_project = false
+DJANGO_SETTINGS_MODULE = kolibri.deployment.default.settings.test
+# Settings for pytest-pythonpath
+python_paths = kolibri/dist
```

Comment:
This ensures that `kolibri/dist` is ALWAYS part of the path environment when invoking pytest -- For now, it is interesting in case it helps us find clutter between our test environment and our distributed environment. But once we start running tests on our build artifacts, it won't be necessary.